### ISSUE TITLE
Pretraining with CutLER

### ISSUE THREAD

**Issue by VGrondin** [facebookresearch/CutLER#15]
Hello, reading your paper was pretty interesting and insightful. 
I was wondering how well an object detector model such as ViTDet can benefit by pretraining with CutLER? 
For instance, from the ViTDet paper, the authors acheive 55.6 APbox and 49.2. APmask (table 5 in Exploring Plain Vision Transformer Backbones for Object Detection), so is it possible to pretrain a ViTDet with CutLER and finetune it in a supervised learning way on COCO to improve detection results?

Thanks again for the great paper.

**Comment by frank-xwang** [comment]
Hi @VGrondin, I believe that using ViTDet pretrained with CutLER could potentially improve performance for semi-/fully supervised learning. I would be interested to hear about any updates you may have on any observed performance gains. Thank you!

**Comment by VGrondin** [comment]
Hi @frank-xwang, thanks for the information. My interest in CutLER mostly lies in the performance improvement on a custom dataset of forest images. I will let you know how it goes, I prepared the pseudo masks using maskcut with a mae backbone pretrained on 100k forest images (example of pseudo mask generated):
![sample0](https://user-images.githubusercontent.com/56001549/225629515-0ba2e451-e996-4f8f-9709-adca0456edeb.jpg)
![sample1](https://user-images.githubusercontent.com/56001549/225629541-dc21e761-6698-4530-9f49-07c3481fd3a9.jpg)

Now I am trying to load vitdet config in cutler, but I am having some compatibility issues between the .yaml config and the recent .py config that vitdet uses. I will let you know how it goes once I find a way to resolve this issue.
